proto/options.proto

```diff
@@ -489,4 +489,7 @@ message Options {
   bool banner_instructions = 55;                                   // Whether to return bannerInstructions in the OSRM serializer response
   float elevation_interval = 56;                                   // Interval for sampling elevation along the route path.  [default = 0.0];
   bool voice_instructions = 57;                                    // Whether to return voiceInstructions in the OSRM serializer response
+  oneof has_isochrone_grid_size {
+    float isochrone_grid_size = 58;                                // Grid size to calculate isochrone
```

Comment:
no need for `oneof` the default value of zero as a sentinel would be fine to note that the person didnt request it